onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -16,6 +16,7 @@
 
 #include <core/graph/constants.h>
 #include <core/framework/path_lib.h>
+#include <core/optimizer/graph_transformer_level.h>
```

Comment:
Can we move the defs to c_api.h?

Reply:
@RyanUnderhill Should this be done?

Reply:
@tracysh I'm not sure if this is useful. The OrtSetSessionGraphOptimizationLevel doesn't match this enum. It's currently limited to disable/basic/all levels of graph optimizations. Is it useful to expose more fine level control?

We'd also lose the 'enum class' stuff as it's limited to pure C there.

Reply:
I don't understand. This is how OrtSetSessionGraphOptimizationLevel sets the level. As you can see we clearly allow 1, 2, and 3

ORT_API_STATUS_IMPL(OrtSetSessionGraphOptimizationLevel, _In_ OrtSessionOptions* options, uint32_t graph_optimization_level) {
  if (graph_optimization_level >= static_cast<uint32_t>(onnxruntime::TransformerLevel::MaxTransformerLevel))
    return OrtCreateStatus(ORT_INVALID_ARGUMENT, "graph_optimization_level is not valid");
  options->value.graph_optimization_level = static_cast<onnxruntime::TransformerLevel>(graph_optimization_level);
  return nullptr;
}